api/routers/cameras.py

```diff
@@ -143,6 +144,23 @@ async def get_camera(camera_id: str):
     return camera
 
 
+def get_first_unused_id(cameras_ids):
```

Comment:
So camera_ids are the existing camera ids? I think that is not obvious from the naming. It would be good to name it `existing_camera_ids` or just add a docstring explaining what the function does and what the parameter contains